<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMyMainWindow</class>
 <widget class="QMainWindow" name="QMyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QMyMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>2</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>8</height>
     </size>
    </property>
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="action_filenew"/>
    <addaction name="action_fileopen"/>
    <addaction name="action_filesave"/>
    <addaction name="action_fileclose"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>8</height>
     </size>
    </property>
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="action_textcut"/>
    <addaction name="action_textcopy"/>
    <addaction name="action_textpaste"/>
    <addaction name="action_textundo"/>
    <addaction name="action_textredo"/>
    <addaction name="action_textclear"/>
    <addaction name="action_textselectall"/>
   </widget>
   <widget class="QMenu" name="menu_M">
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>8</height>
     </size>
    </property>
    <property name="title">
     <string>格式(M)</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="font">
      <font>
       <family>Viner Hand ITC</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>界面语言</string>
     </property>
     <addaction name="action_English"/>
     <addaction name="action_Chinese"/>
    </widget>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>在这里输入</string>
     </property>
    </widget>
    <addaction name="action_bold"/>
    <addaction name="action_textitalic"/>
    <addaction name="action_textunderline"/>
    <addaction name="action_textshow"/>
    <addaction name="menu"/>
    <addaction name="menu_2"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_M"/>
  </widget>
  <widget class="QToolBar" name="filetoolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>5</width>
     <height>8</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>File</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_fileopen"/>
   <addaction name="action_filenew"/>
   <addaction name="action_filesave"/>
  </widget>
  <widget class="QToolBar" name="actiontoolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>5</width>
     <height>8</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>Action</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_textcut"/>
   <addaction name="action_textcopy"/>
   <addaction name="action_textpaste"/>
   <addaction name="action_textundo"/>
   <addaction name="action_textredo"/>
   <addaction name="action_textselectall"/>
   <addaction name="action_textclear"/>
  </widget>
  <widget class="QToolBar" name="texttoolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>5</width>
     <height>8</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>Edit</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_bold"/>
   <addaction name="action_textitalic"/>
   <addaction name="action_textunderline"/>
  </widget>
  <widget class="QToolBar" name="LanguagetoolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>5</width>
     <height>8</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>Language</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_English"/>
   <addaction name="action_Chinese"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <action name="action_fileopen">
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/001.GIF</normaloff>:/images/images/images/001.GIF</iconset>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_fileclose">
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/132.bmp</normaloff>:/images/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭文件</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="action_textcut">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/200.bmp</normaloff>:/images/images/images/200.bmp</iconset>
   </property>
   <property name="text">
    <string>剪贴</string>
   </property>
   <property name="toolTip">
    <string>剪贴</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_textpaste">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/204.bmp</normaloff>:/images/images/images/204.bmp</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_textcopy">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/202.bmp</normaloff>:/images/images/images/202.bmp</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/500.bmp</normaloff>:/images/images/images/500.bmp</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_textitalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/502.bmp</normaloff>:/images/images/images/502.bmp</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_textunderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/504.bmp</normaloff>:/images/images/images/504.bmp</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_textshow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/114.bmp</normaloff>:/images/images/images/114.bmp</iconset>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
   <property name="toolTip">
    <string>显示图标名称</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+W</string>
   </property>
  </action>
  <action name="action_textclear">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/212.bmp</normaloff>:/images/images/images/212.bmp</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空文字</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+B</string>
   </property>
  </action>
  <action name="action_textundo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/206.bmp</normaloff>:/images/images/images/206.bmp</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>返回上一状态</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_textredo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/208.bmp</normaloff>:/images/images/images/208.bmp</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="toolTip">
    <string>将&quot;撤销&quot;撤销</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_textselectall">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/116.bmp</normaloff>:/images/images/images/116.bmp</iconset>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>选中全部文字</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_filenew">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/100.bmp</normaloff>:/images/images/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_filesave">
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/104.bmp</normaloff>:/images/images/images/104.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_English">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/timg2.jpg</normaloff>:/images/images/images/timg2.jpg</iconset>
   </property>
   <property name="text">
    <string>英语</string>
   </property>
   <property name="toolTip">
    <string>显示英语</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="action_Chinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="QMyMainWindow.qrc">
     <normaloff>:/images/images/images/CN.jpg</normaloff>:/images/images/images/CN.jpg</iconset>
   </property>
   <property name="text">
    <string>汉语</string>
   </property>
   <property name="toolTip">
    <string>显示中文</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="QMyMainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_textundo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_textredo</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_textcut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_textcopy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_textpaste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_textclear</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_textselectall</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>action_textundo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_fileclose</sender>
   <signal>triggered()</signal>
   <receiver>QMyMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>redoAvailable(bool)</signal>
   <receiver>action_textredo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_textcopy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_textcut</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>action_textpaste</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
